foundation/foundation-upgrade/src/test/resources/specifications/v1_9_0/airflow-dockerfile-migration.feature

```diff
@@ -0,0 +1,11 @@
+Feature: Migration Airflow Dockerfile
+
+  Scenario: Update the Airflow Dockerfile to pull the community Docker image
+    Given a Dockerfile is referencing the airflow image from aissemble-airflow
+    When the 1.9.0 Airflow Docker image migration executes
+    Then the image will pull the community docker Airflow image
+
+  Scenario: Do not update the Airflow Dockerfile when the file is already pulling the community Airflow Docker image
+    Given a Docker file is referencing the community airflow image
```

Comment:
S: I would think we'd want to test with a non-airlfow Dockerfile. Something like the PDP build that still pulls a baseline image.

Reply:
I left this as is to test the negative test case. However, I added a new BDD scenario to test a non Airflow Dockerfile.